scipy/stats/_qmc.py

```diff
@@ -799,7 +799,12 @@ def integers(
         else:
             sample = self.random(n=n)
 
-        sample = scale(sample, l_bounds=l_bounds, u_bounds=u_bounds)
+        # scaling, no check of hypercube as sample are generated by engine
```

Comment:
Can this go back to using `scale` now that `MultinomialQMC` and  `MultivariateNormalQMC` won't use this?

Reply:
@tupui?

Reply:
Forgot! will fix this now.